04/app.js

```diff
@@ -5,4 +5,57 @@ const menuItems = [
     {text: 'start', url: '/'},
     {text: 'galeria', url: '/gallery'},
     {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+];
+
+const nav = document.querySelector('nav');
+console.log(nav);
+const newUl = document.createElement('ul');
+
+nav.appendChild(newUl);
+
+
+for (let i=0; i<3; i++) {
```

Comment:
Lepiej zrobić `i<menuItems.length`, aby dodanie elementu do tej tablicy nie powodowało zmian w pętli `for`